oshi-core/src/main/java/oshi/hardware/platform/unix/UnixDisplay.java

```diff
@@ -53,6 +53,6 @@ public final class UnixDisplay extends AbstractDisplay {
      * @return An array of Display objects representing monitors, etc.
      */
     public static List<Display> getDisplays() {
-        return Xrandr.getEdidArrays().stream().map(UnixDisplay::new).collect(Collectors.toList());
+        return Xrandr.getEdidArrays().stream().parallel().map(UnixDisplay::new).collect(Collectors.toList());
```

Comment:
This could in theory improve performance but at the cost of giving users a random order of their displays each time they ran the method.  I'd vote no.